synapse/lib/cell.py

```diff
@@ -706,7 +707,9 @@ def _loadCellYaml(self, *path):
         return await self.getCellApi(link, user, path)
 
     async def getCellApi(self, link, user, path):
-        return await self.cellapi.anit(self, link, user)
+        capi = await self.cellapi.anit(self, link, user)
```

Comment:
Even this can't be used b/c we have Cell implementation which override getCellApi